### ISSUE TITLE
請問這類的驗證碼的樣本數和配置

### ISSUE THREAD

**Issue by areckkimo** [kerlomz/captcha_trainer#106]
請教一下，目前驗證碼的特徵有： 斜體字 / 前景干擾 / 背景干擾 / 多種字體
![image](https://user-images.githubusercontent.com/9306247/192102675-f56fb9e7-6632-4d0f-817f-bffd39659e8a.png)

目前配置
Label Num: 5
CNN Layer: CNN3 
Recurrent Layer:  GRU
UnitsNum: 64
Loss Function： CTC
Optimizer: RAdam
Learning Rate: 0.001
Category: Numeric

Epoch已經1433但是Accuracy仍然是0，訓練樣本數是700張，用來驗證有300張

請問可以調整配置，讓Accuracy上來

**Comment by kerlomz** [comment]
增加样本到5千张，网络选CNNX，应该可以达到80-90，1万张应该可以95以上

**Comment by areckkimo** [comment]
> 增加样本到5千张，网络选CNNX，应该可以达到80-90，1万张应该可以95以上

謝謝
請問CNNX和CNN3差別
